.github/workflows/continuous-integration.yml

```diff
@@ -7,6 +7,14 @@ on:
 
 
 jobs:
+  run-benchmark:
+    name: run benchmark
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: jasonwilliams/criterion-compare-action@move_to_actions
```

Comment:
This really doesn't look like something that I want to run on every push (it also does nothing on master, so let's not).

Maybe this should be in a different workflow, with a manually triggered condition (I don't know what conditions are available to us, though).